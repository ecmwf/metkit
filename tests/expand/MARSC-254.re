retrieve,
    type=ob,
    date=-3,
    time=00,
    range=1439,
    target=data.reference 