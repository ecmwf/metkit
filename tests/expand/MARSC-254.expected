RETRIEVE,
    CLASS      = OD,
    TYPE       = OB,
    STREAM     = OPER,
    EXPVER     = 0001,
    REPRES     = BU,
    OBSTYPE    = 1,
    DATE       = 20250414,
    TIME       = 0000,
    RANGE      = 1439,
    DOMAIN     = G,
    TARGET     = data.reference,
    DUPLICATES = KEEP 